frontend/src/static/css/includes/form_controls/_buttons.scss

```
@@ -1,19 +1,19 @@
-$success-color: #00a28b;
-$warning-color: #e09f1f;
-$danger-color: #de623b;
+$success-color: var(--cinemata-coral-reef-500);
+$warning-color: var(--cinemata-amber-500);
+$danger-color: var(--cinemata-sunset-horizon-500);
 
-$brand-color: #008ede;
+$brand-color: var(--cinemata-strait-blue-400);
```

Stylelint (17.11.1)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)